src/main/java/com/ioet/bpm/skills/domain/converters/CategoryConverter.java

```diff
@@ -0,0 +1,54 @@
+package com.ioet.bpm.skills.domain.converters;
+
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBTypeConverter;
+import com.ioet.bpm.skills.domain.Category;
+import com.ioet.bpm.skills.domain.converters.exception.ConverterException;
+
+public class CategoryConverter implements DynamoDBTypeConverter<String, Category> {
```

Comment:
is this class still needed? 
I think it is not.

Reply:
Probably you can delete the whole converters package as well as the domain package in the test folder